frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar/QuestionDetails.tsx

```diff
@@ -0,0 +1,125 @@
+import cx from "classnames";
+import { useState } from "react";
+import { c, t } from "ttag";
+
+import { SidesheetCardSection } from "metabase/common/components/Sidesheet";
+import DateTime from "metabase/components/DateTime";
+import Styles from "metabase/css/core/index.css";
+import { getUserName } from "metabase/lib/user";
+import { QuestionPublicLinkPopover } from "metabase/sharing/components/PublicLinkPopover";
+import { Box, Flex, Icon, Text } from "metabase/ui";
+import type Question from "metabase-lib/v1/Question";
+
+import SidebarStyles from "./QuestionInfoSidebar.module.css";
+
+export const QuestionDetails = ({ question }: { question: Question }) => {
+  const lastEditInfo = question.lastEditInfo();
+  const createdBy = question.getCreator();
+  const createdAt = question.getCreatedAt();
+
+  return (
+    <>
+      <SidesheetCardSection title={t`Creator and last editor`}>
+        {lastEditInfo && (
+          <Flex gap="sm" align="center">
+            <Icon name="ai" />
+            <Text>
+              {c("{0} is a date/time and ${1} is a person's name").jt`${(
+                <DateTime
+                  unit="day"
+                  value={lastEditInfo.timestamp}
+                  key="date"
+                />
+              )} by ${getUserName(lastEditInfo)}`}
+            </Text>
+          </Flex>
+        )}
+
+        <Flex gap="sm" align="center">
+          <Icon name="pencil" />
+          <Text>
+            {c("{0} is a date/time and ${1} is a person's name").jt`${(
+              <DateTime unit="day" value={createdAt} key="date" />
+            )} by ${getUserName(createdBy)}`}
+          </Text>
+        </Flex>
+      </SidesheetCardSection>
+      <SidesheetCardSection title={t`Saved in`}>
```

Comment:
A context string here would probably help in a lot of languages, maybe something like "c("This is a heading. Below it is the name of a collection that a question is saved in.")